README.md

```diff
@@ -26,6 +26,15 @@ xFormers is:
 - **Research first**: xFormers contains bleeding-edge components, that are not yet available in mainstream libraries like pytorch.
 - **Built with efficiency in mind**: Because speed of iteration matters, components are as fast and memory-efficient as possible. xFormers contains its own CUDA kernels, but dispatches to other libraries when relevant.
 
+## Supported Platforms
+
+xFormers is optimized for and tested on systems with NVIDIA GPUs, including:
+
+- Linux with x86-64 architecture
+- Windows with x86-64 architecture
+
+Apple M1/M2 chips (arm64 architecture) are currently not supported. We aim to add support in a future release.
```

Comment:
We don't have a plan to support apple chips at the moment. This would require a somehow different expertise, and most importantly that's not something that would serve our internal users so we don't plan to prioritize it
```suggestion
Apple M1/M2 chips (arm64 architecture) are currently not supported.
```